src/helpers/utils.ts

```diff
@@ -48,3 +49,16 @@ export async function getEmailAddresses(email: string): Promise<{ address: strin
 export async function getUniqueEmails(): Promise<{ email: string }[]> {
   return await db.queryAsync('SELECT DISTINCT email FROM subscribers WHERE verified > 0');
 }
+
+// RFC5322 standard, does support most format, but not all
+// See test files for list of tested formats
+export function isValidEmail(input: string) {
+  return new RegExp(
+    /^(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])$/,
+    'gi'
+  ).test(input);
+}
+
+export function isValidAddress(input: string) {
```

Comment:
why do you need this wrapper?

Reply:
`isValidAddress` only calls the function `isAddress`, so you can remove it and export just `isAddress` in case you want to hide it in the `utils` file. But you can consider using `import { isAddress } from '@ethersproject/address';` directly in place, you use it.